.github/workflows/build-rpm.yml

```diff
@@ -64,6 +64,10 @@ jobs:
             convert2rhel/__init__.py
           retention-days: 1
 
+      - name: Trigger Manpage Generation
+        run: |
+          oamg/convert2rhel/.github/workflows/generate_manpage.yml@main
+
```

Comment:
To get this to work `generate_manpage.yml` needs to be a runnable workflow. But don't think it's necessary to run it here tbh, we can keep to just running it on every PR